Source/WTF/wtf/StdLibExtras.h

```diff
@@ -205,6 +205,26 @@ template<size_t divisor, typename T> inline constexpr T* roundUpToMultipleOf(T*
     return reinterpret_cast<T*>(roundUpToMultipleOf<divisor>(reinterpret_cast<size_t>(x)));
 }
 
+template<typename T, typename U>
+inline constexpr T roundUpToMultipleOfNonPowerOfTwo(U divisor, T x)
```

Comment:
Why does the divisor come first?
Should we use SFINAE to open enable this function for integral types?

Reply:
> Why does the divisor come first? 

Theres numerous similar functions, where the divisor does come first. Also the template parameters T, U are in that order in other functions.

> Should we use SFINAE to open enable this function for integral types?

Well should we ? :) If a thing is useful to mention as a review comment, it should be useful to spend the time and expand the comment with a sentence of a rationale.
I'm not sure what that means, SFINAE to open enable for integral types. Given that this function is rather similar to numerous other functions that do the exact same thing around it, and it works for this use-case, I'd say not in this patch? Maybe all the functions can be improved later on.

Note: the Checked and non-Checked cannot be supported by the same template, if that's what you mean. The checked variant currently has to check the overflow during the if and during the mod. Checked could be fixed for mod, but elaborate framework removing the if wouldn't let the same template be used.